cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/internal/BootstrapCoordinator.scala

```diff
@@ -123,6 +123,7 @@ private[akka] class BootstrapCoordinator(discovery: ServiceDiscovery,
 
   private var lastContactsObservation: Option[ServiceContactsObservation] = None
   private var seedNodesObservations: Map[ResolvedTarget, SeedNodesObservation] = Map.empty
+  private var selfUri: Uri = _ // set via InitiateBootstrapping
```

Comment:
Instead of adding another `var` here, could we pass this value in as an extra parameter to `bootstrapping`?